src/__tests__/GobanCanvas.test.ts

```diff
@@ -356,15 +374,6 @@ describe("onTap", () => {
                 stones: "babbbabbbbba",
```

Comment:
I noticed there is a lot of redundancy in this message.  I wonder what's going on in `toggleMetaGroupRemoval()` that is causing this...